GitTabExpansion.ps1

```diff
@@ -74,6 +74,12 @@ function script:gitBranches($filter, $includeHEAD = $false) {
         foreach { $prefix + $_ }
 }
 
+function script:gitTags($filter) {
+    git tag |
+        where { $_ -like "$filter*" } |
+        foreach { $_ }

```

Comment:
I'm asking specifically about the line `foreach { $_ }`.

Try this: remove that line, try TAB-completion on tags, and see if it still works.